### PR TITLE
[DEVOPS-1195] Adding files required to build the docker image

### PR DESCRIPTION
**Pull request by taksan**

### CODE REVIEW

File: docker/start.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+	WAR_PATH=$(pwd)
+	cd $TASKBOARD_DATA
+	application
+}
+
+application() {
+	TS=$(date +%Y%m%d%H%M%S)
+	cp $TASKBOARD_DATA/taskboard.log $TASKBOARD_DATA/taskboard.log.$TS || echo "No log to rotate."
+	java $TASKBOARD_XMX_XMS -cp $TASKBOARD_DATA:$WAR_PATH/taskboard.war -Dobjective.taskboard.config.enableBootstrapTasks=true org.springframework.boot.loader.WarLauncher --server.port=8082| tee  $TASKBOARD_DATA/taskboard.log
```
Comment (herbert-venancio) [inline docker/start.sh]: In `Dockerfile` it:
```
# Http port
EXPOSE 8080
```
But here it uses port 8082

File: docker/start.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+if [ "${1:0:1}" != '-' ]; then
+	exec "$@"
+fi
+
+TASKBOARD_DATA=/opt/taskboard_config
+
+main() {
+	cd $(dirname $(readlink -f $0))
```
Comment (herbert-venancio) [inline docker/start.sh]: Can `$(dirname ...)` be directly assigned to `WAR_PATH`?

File: docker/start.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+if [ "${1:0:1}" != '-' ]; then
```
Comment (herbert-venancio) [inline docker/start.sh]: if I run:
```bash
./start.sh echo "Hello world"
```
it prints `Hello world`.
But in practice, what is this used for?

File: docker/taskboard-service [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  *)
+    echo "Usage: /etc/init.d/taskboard-service {start|stop|restart|update}"
+    exit 1
+    ;;
+  esac
+}
+
+do_start() {
+    docker rm -f $NAME 2>/dev/null
+    docker run -d --name $NAME -v $DOCKER_WORKSPACE -p 8089:8082 $IMG
```
Comment (herbert-venancio) [inline docker/taskboard-service]: In `Dockerfile` it:
```
# Http port
EXPOSE 8080
```
But here it uses port 8082 and maps to port 8089

File: docker/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8
+
+MAINTAINER DevOps @ Objective <devops@objective.com.br>
+
+ENV TASKBOARD_HOME=/opt/taskboard 
+
+# Http port
+EXPOSE 8080
+
+RUN set -x \
```
Comment (herbert-venancio) [inline docker/Dockerfile]: Why `set -x`?

File: docker/start.sh (hunk 1 above)
Comment (taksan) [reply docker/start.sh]: better fix

**Pull request merged**